SimplenoteUITests/SimplenoteUITestsNoteEditor.swift

```diff
@@ -276,7 +277,6 @@ class SimplenoteUISmokeTestsNoteEditor: XCTestCase {
         NoteEditorAssert.textViewWithExactLabelsShownOnce(labels: ["* Asterisk1", "* Asterisk2", "* Asterisk3"])
 
         trackStep()
-        NoteEditor.markdownEnable()
```

Comment:
> Edit: besides this, moving the call to enable Markdown in the note to an earlier stage (as done for all other tests) seems to finally solve the issue. 🧙

Maybe this has to do with us trying to swipe immediately after, but the screen doing a little bounce animation when Markdown gets enabled? _Animations are the crux of UI automation_.


https://user-images.githubusercontent.com/1218433/126108050-138d8791-e267-4667-a5fa-c4ae2fbaa7b3.mov

Reply:
> Animations are the crux of UI automation.

Yes, I think this was the case here. Thank you for the review @mokagio, as always! 🙂

Reply:
Likewise. Thank you for caring and for keeping this in good shape!